terraform/deploy/infra/modules.tf

```diff
@@ -29,13 +29,6 @@ module analytical_env_vpc {
     "git-codecommit"
   ]
 
-  custom_vpce_services = [
-    {
-      key          = "proxy_vpc_endpoint"
-      service_name = data.terraform_remote_state.internet_egress.outputs.internet_proxy_service.service_name
-      port         = 3128
-    }
-  ]
```

Comment:
This sets up the proxy endpoint, the `service_name` has to come from somewhere and I assume that is `internet_egress`

Reply:
No it doesn't set up the proxy endpoint, any more. That's kinda the point of this PR. The proxy endpoint is deployed in another manner.